tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "ES2019",
     "module": "commonjs",
-    "lib": ["es2017","es2019.array", "DOM"],
+    "lib": ["ESNext", "DOM"],
```

Comment:
is ESNext 100% safe here?

Reply:
ESNext uses all the downstream versions like ES2019 and ES2017, so yes.